test/e2e/edu-5490-helpers.js

```diff
@@ -0,0 +1,150 @@
+// Also actually uses the data
+function completeBenefitSelection(client, data) {
+  client.click(`input[value="${data.benefit}"]`);
+}
+
+function completeBenefitRelinquishment(client, data) {
+  const date = data.benefitsRelinquishedDate.split('-');
+  client
+    .selectDropdown('root_benefitsRelinquishedDateMonth', date[1])
+    .selectDropdown('root_benefitsRelinquishedDateDay', date[2])
+    .fill('input[name="root_benefitsRelinquishedDateYear"]', date[0]);
+}
+
+function completeBenefitHistory(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const { previousBenefits } = data;
+    client
+      .clickIf('input[name="root_previousBenefits_view:noPreviousBenefits"]', previousBenefits['view:noPreviousBenefits'])
+      .clickIf('input[name="root_previousBenefits_disability"]', previousBenefits.disability)
+      .clickIf('input[name="root_previousBenefits_dic"]', previousBenefits.dic)
+      .clickIf('input[name="root_previousBenefits_chapter31"]', previousBenefits.chapter31);
+
+    if (previousBenefits['view:ownServiceBenefits']) {
+      client
+        .click('input[name="root_previousBenefits_view:ownServiceBenefits"]')
+        .fill('input[name="root_previousBenefits_ownServiceBenefits"]', previousBenefits.ownServiceBenefits);
+    }
+
+    if (previousBenefits['view:claimedSponsorService']) {
+      client
+        .click('input[name="root_previousBenefits_view:claimedSponsorService"]')
+        .clickIf('input[name="root_previousBenefits_chapter35"]', previousBenefits.chapter35)
+        .clickIf('input[name="root_previousBenefits_chapter33"]', previousBenefits.chapter33)
+        .clickIf('input[name="root_previousBenefits_transferOfEntitlement"]', previousBenefits.transferOfEntitlement)
+        .fill('input[name="root_previousBenefits_veteranFullName_first"]', previousBenefits.veteranFullName.first)
+        .fill('input[name="root_previousBenefits_veteranFullName_middle"]', previousBenefits.veteranFullName.middle)
+        .fill('input[name="root_previousBenefits_veteranFullName_last"]', previousBenefits.veteranFullName.last)
+        .selectDropdown('root_previousBenefits_veteranFullName_suffix', previousBenefits.veteranFullName.suffix)
+        .fill('input[name="root_previousBenefits_veteranSocialSecurityNumber"]', previousBenefits.veteranSocialSecurityNumber);
+    }
+
+    client.fill('input[name="root_previousBenefits_other"]', previousBenefits.other);
+  }
+}
+
+function completeSponsorService(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const activeDuty = `root_currentlyActiveDuty${data.currentlyActiveDuty ? 'Yes' : 'No'}`;
+    const felony = `root_outstandingFelony${data.outstandingFelony ? 'Yes' : 'No'}`;
+    client
+      .fill('input[name="root_serviceBranch"]', data.serviceBranch)
+      .click(`input[name="${activeDuty}"]`)
+      .click(`input[name="${felony}"]`);
+  }
+}
+
+function completeSecondaryContact(client, data, onlyRequiredFields) {
```

Comment:
I'm guessing other forms will need this as well, but I guess it's OK to leave it here and pull it into shared code when needed (assuming we don't forget about it).